### PR TITLE
Add L1 image decoder DALI test

### PR DESCRIPTION
**Pull request by JanuszL**
## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- adds L1 image decoder DALI test

### Affected modules and functionalities:
- tests

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [x] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [x] Benchmark
    - TL1_decoder_perf
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

File: qa/TL1_decoder_perf/test.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+test_body() {
+  # SPEC:DA-11356-002_v04
+  if [ "$(uname -p)" == "x86_64" ]; then
+    # Hooper
+    MIN_PERF=19000;
+    python hw_decoder_bench.py --width_hint 6000 --height_hint 6000 -b 408 -d 0 -g gpu -w 10 -t 10000 -i ${DALI_EXTRA_PATH}/db/single/jpeg -p rn50 -j 70 --hw_load 0.12 | tee ${LOG}
+  else
+    # GraceHooper
```
[the remark this answers is not in the document]
Comment (JanuszL) [reply qa/TL1_decoder_perf/test.sh:19]: Fixed

File: qa/TL1_decoder_perf/test.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+LOG="dali.log"
+function CLEAN_AND_EXIT {
+    rm -rf ${LOG}
+    exit $1
+}
+
+test_body() {
+  # SPEC:DA-11356-002_v04
+  if [ "$(uname -p)" == "x86_64" ]; then
+    # Hooper
```
[the remark this answers is not in the document]
Comment (JanuszL) [reply qa/TL1_decoder_perf/test.sh:15]: Fixed

**Review by awolant — APPROVED** [review]

File: qa/TL1_decoder_perf/test.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (JanuszL) [reply qa/TL1_decoder_perf/test.sh]: Forgot to push. Now it is done.

File: qa/TL1_decoder_perf/test.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (JanuszL) [reply qa/TL1_decoder_perf/test.sh]: Forgot to push. Now it is done.

**Pull request merged**